### PR TITLE
Mobile and Desktop versions 1

### PR DESCRIPTION
**Pull request by Franz940**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>🔥 Fireplace Palace</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
```
Comment (tompsoc) [inline index.html:15]: I would probably wrap the top level of the application in a div with a class of wrapper. That way you don't need to turn the body into a grid/flex container :)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    <!-- Header Section -->
+    <header class="header-container">
+      <h1>🔥 Fireplace Palace</h1>
+    </header>
+
+    <!-- Hero Section -->
+    <section id="hero" class="hero-section">
```
Comment (tompsoc) [inline index.html]: Do you need the id?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      <img
+        src="/images/hero-mobile.png"
```
Comment (tompsoc) [inline index.html]: You could use srcset to pull in different images on mobile/desktop? :)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>🔥 Fireplace Palace</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <!-- Header Section -->
+    <header class="header-container">
+      <h1>🔥 Fireplace Palace</h1>
+    </header>
+
```
Comment (tompsoc) [inline index.html:21]: You could wrap your sections with main maybe?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+      </div>
+    </section>
+
+    <!-- Footer Section -->
+    <footer class="footer">
+      <div class="colunms-container">
+        <div class="footer-column">
+          <p>Find us on:</p>
+          <nav class="footer-nav">
+            <a href="https://www.facebook.com" class="link"> Facebook </a>
```
Comment (tompsoc) [inline index.html]: This links in the footer might make more sense semantically as a list? :)

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,154 @@
+html,
+body {
```
Comment (tompsoc) [inline styles.css:2]: I think it might be useful to set box-sizing: border-box, that's a common default :)

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+  width: 100%;
+  font-size: 16px;
```
Comment (tompsoc) [inline styles.css:5]: I usually set font-size of 10px on the html/body, then I use rems in the rest of my code. Generally rems are recommended but I'm easy if you'd like to keep px!

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+  width: 100%;
+  height: 300px;
+  object-fit: cover;
+}
+.hero-headline {
+  margin: 0;
+  padding: 0;
+}
+.hero-description {
+  background-color: #f1911b;
```
Comment (tompsoc) [inline styles.css:31]: If there are 1 or 2 key colours feel free to use css variables. Or leave in place - I'm easy :)

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+  display: flex;
+  flex-direction: column;
+}
+.footer-nav {
+  display: flex;
+  flex-direction: column;
+}
+.link {
+  text-decoration: none;
+  color: black;
```
Comment (tompsoc) [inline styles.css]: I'd probably try to keep to using hex consistently :)

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+}
+.footer-colunm {
+  display: flex;
+  flex-direction: column;
+}
+.footer-nav {
+  display: flex;
+  flex-direction: column;
+}
+.link {
```
Comment (tompsoc) [inline styles.css]: I'd probably style the footer links as "a" elements under the footer nav or I'd consider slightly more specific naming :)

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+  display: flex;
+  justify-content: space-between;
+  font-size: 22px;
+}
+.footer-colunm {
+  display: flex;
+  flex-direction: column;
+}
+.footer-nav {
+  display: flex;
```
Comment (tompsoc) [inline styles.css]: If you made the links a list of links you wouldn't need to use flex for the footer links. You could just use a list and display block/inline when you want to go from stacked list to inline links. I'm easy on this one :)

**Review by tompsoc — CHANGES_REQUESTED** [review]
This is brilliant dude :) nice use of semantic HTML and clean CSS. You've used some elements I don't think I ever did! I put some comments, have a read and ask any questions on the comment if something doesn't make sense.

File: index.html (hunk 2 above)
Comment (Franz940) [reply index.html]: Not really actually, I think I added to make it semantic but I haven't to others elements so I will remove it.

**Pull request merged**